backend/src/main/java/gov/cdc/usds/simplereport/service/PatientSelfRegistrationLinkService.java

```diff
@@ -28,9 +30,21 @@
 
   public PatientSelfRegistrationLink getPatientRegistrationLink(String patientRegistrationLink)
       throws InvalidPatientSelfRegistrationLinkException {
-    return prlrepo
-        .findByPatientRegistrationLink(patientRegistrationLink)
-        .orElseThrow(InvalidPatientSelfRegistrationLinkException::new);
+    PatientSelfRegistrationLink link =
+        prlrepo
+            .findByPatientRegistrationLinkIgnoreCase(patientRegistrationLink)
+            .orElseThrow(InvalidPatientSelfRegistrationLinkException::new);
+
+    Organization org =
+        link.getOrganization() != null
```

Comment:
**suggestion:** This ternary shouldn't be necessary since `PatientSelfRegistrationLink.getOrganization` performs the inverse check for `facility != null`. Do you think we're safe to remove it?

Reply:
Ah right, didn't see that. Fixed up!